src/components/entityListItem/entityListItem.scss

```diff
@@ -14,9 +15,11 @@
     border-radius: 50%;
   }
   .entityTitle {
-    @include typography.text-preset-4-bold;
+    // @include typography.text-preset-4-bold;
```

Comment:
You can remove the comments. Later, when we will be older, and possibly wiser, we won't know why the lines were commented.